types/ember/test/array-proxy.ts

```diff
@@ -9,7 +9,7 @@ proxy.get('firstObject'); // 'amoeba'
 
 const overridden = Ember.ArrayProxy.create({
     content: Ember.A(pets),
-    objectAtContent(idx) {
+    objectAtContent(idx: number) {
```

Comment:
Ah, I totally missed that that was happening before! (And didn't stop to reflect on why I'd unthinkingly made that change in this file). While it's pretty cool that that was working, I'd agree that sticking as close to the vanilla object model as possible will hopefully yield us the best results in the long run :)